etc/ci/buildbot_steps.yml

```diff
@@ -72,7 +72,7 @@ linux-rel-nogate:
   - ./mach clean-nightlies --keep 3 --force
   - env CC=gcc-5 CXX=g++-5 ./mach build --release
   - python ./etc/ci/chaos_monkey_test.py
-  - env CC=gcc-5 CXX=g++-5 bash ./etc/ci/mutation_test.sh
+  - env CC=gcc-5 CXX=g++-5 RUSTFLAGS= bash ./etc/ci/mutation_test.sh
```

Comment:
Maybe `RUSTFLAGS="--cap-lints warn"`? That way we also allow mutations in crates that don't allow warnings via `#![deny()]` blocks.